torchgeo/trainers/classification.py

```diff
@@ -44,7 +44,11 @@ class ClassificationTask(LightningModule):  # type: ignore[misc]
     """
 
     def config_model(self) -> None:
-        """Configures the model based on kwargs parameters passed to the constructor."""
+        """Configures the model based on kwargs parameters passed to the constructor.
+
+        .. versionchanged:: 0.5
+           Added *freeze_backbone* parameter.
```

Comment:
This function has no parameters

Reply:
Should I remove the version changed completely then? Or should I change the comment to something like `Added ability freeze_backbone` ?

Reply:
Yes, remove it. I'm actually planning on getting rid of this function entirely in a refactor, or at least making it private